docs/user-guide/layer-settings.md

```diff
@@ -1,120 +1,187 @@
 
-* **Click** on the **Create new style** button <img src="../img/button/style_editor_new_style_button.jpg" class="ms-docbutton"/> if you want to create your style from scratch.   [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) shows you some style templates from which you can start your customization. Those templates can be:
+### Create a new style 
 
-    * ***CSS - Cascading Style Sheet*** (a language used for describing the presentation of a document written in a markup language like the HTML)
+It is possible to create a new style with a click on the <img src="../img/button/style_editor_new_style_button.jpg" class="ms-docbutton"/> button. At this stage the user can choose between different types of template from which the customization will start:
 
-    * ***SLD - Styled Layer Descriptor*** (an XML schema specified by the [Open Geospatial Consortium (OGC)](http://www.opengeospatial.org/) for describing the appearance of map layers)
+* *CSS - Cascading Style Sheet* (a language used for describing the presentation of a document written in a markup language like the HTML)
 
-<img src="../img/layer-settings/style_editor_add_style_template.jpg" class="ms-docimage"  style="max-width:500px;">
+* *SLD - Styled Layer Descriptor* (an XML schema specified by the [Open Geospatial Consortium OGC](http://www.opengeospatial.org/) for describing the appearance of map layers)
 
-* **Select** a style template
+<img src="../img/layer-settings/style_editor_add_style_template.jpg" class="ms-docimage" style="max-width:500px;">
 
-* **Click** on the **Add selected template to list of styles** button <img src="../img/button/style_editor_add_style_button.jpg" class="ms-docbutton"/>
+!!!note
+    The availability of the style formats depends, firstly, from the [GeoServer](http://geoserver.org/). [MapStore](https://mapstore.geo-solutions.it/mapstore/#/), by default, will add all the supported format that the server provides.
 
-* **Enter** a *Title* (and an *Abstract* if you want to) for your style
+Once the new style is chosen, with a click on the <img src="../img/button/style_editor_add_style_button.jpg" class="ms-docbutton"/> button the following window opens:
 
 <img src="../img/layer-settings/style_editor_new_style_name.jpg" class="ms-docimage"  style="max-width:500px;">
 
-* **Click** on *Save*
+In here the user can set the *Title* and the *Abstract* (optional), and through the **Save** button the new style will be automatically added to the styles list.
 
-* Now you can see your new style in the styles' list and **Edit** it through the editor
+### Edit an existing style
+
+Existing styles can be edited clicking on the <img src="../img/button/style_editor_edit_button.jpg" class="ms-docbutton"/> button. The page that opens allows the user to customize the style through an xml text:
```

Comment:
... The page that opens allows the user to customize the style in the related format: